Foundation/NSURLSession/HTTPMessage.swift

```diff
@@ -141,12 +141,17 @@ extension URLSessionTask {
 
 extension URLSessionTask._HTTPMessage {
     var headersAsDictionary: [String: String] {
-        var result: [String: String] = [:]
-        headers.forEach {
-            result[$0.name] = $0.value
-        }
-        return result
-    }
+		var result: [String: String] = [:]
```

Comment:
Drive-by nit: these lines should use the same indenting as the rest of the file (4 spaces, not tab).